packages/build-utils/src/executors/client-generator/client-generator.ts

```diff
@@ -32,21 +35,47 @@ export default async function generateClients(options: ClientGeneratorSchemaType
     throw new Error('invalid options passed to client-generator executor');
   }
 
-  const targetPath = context.projectsConfigurations?.projects[context.projectName]?.root;
-  Object.keys(options.specs).forEach((specName) => {
-    let specPath = options.specs[specName];
-    if (specPath.indexOf('http://') < 0 && specPath.indexOf('https://') < 0) {
-      specPath = path.join(targetPath, specPath);
+  const projectConfig = context.projectsConfigurations?.projects[context.projectName];
+
+  // Extract client name from project name
+  if (!projectConfig || !projectConfig.name) {
+    throw new Error('Project name is required to generate client name');
+  }
+
+  const packagePath = projectConfig.root;
+
+  const packageName = projectConfig.name.split('/')[1]; // "rbac-client"
```

Comment:
@charlesmulder not blocking -but as another follow up instead of doing all these gymnastics - I wonder if we put another field in each project.json for the clientName so it is explicitly set in each client (like we do with the input/output now).